ReactWindows/ReactNative.Shared/Bridge/ReactContext.cs

```diff
@@ -394,6 +394,11 @@ public void RunOnJavaScriptQueueThread(Action action)
             _reactInstance.QueueConfiguration.JavaScriptQueue.Dispatch(action);
         }
 
+        /// <summary>
+        /// The native modules queue.
+        /// </summary>
+        public IActionQueue NativeModulesQueue => _reactInstance.QueueConfiguration.NativeModulesQueue;
+
```

Comment:
Do we have to make this public?

Reply:
👍

Reply:
I switched it to `internal` at your request. Do you have any particular concerns about making it `public`? One use case I can imagine is if somebody wants to pick between a custom queue and the native modules queue during module construction time. I suppose in the latter case they can call `base.ActionQueue`.

I know it's probably best to default things to non-public. The ActionQueue stuff seems like a pretty fundamental public concept for RNW which is why I considered making it public. It would be annoying if somebody wanted to get ahold of the native modules queue but couldn't because we made it internal.

Reply:
My general feeling is let's keep it internal until we need to make it public. Especially for a framework that already has public API surface issues, I'm wary of making the problem even worse.

---
In reply to: [169538653](https://github.com/Microsoft/react-native-windows/pull/1650#discussion_r169538653) [](ancestors = 169538653)

Reply:
That's fine with me.

---
In reply to: [169584942](https://github.com/Microsoft/react-native-windows/pull/1650#discussion_r169584942) [](ancestors = 169584942,169538653)